test/gitea_test.exs

```diff
@@ -131,6 +131,7 @@ defmodule GiteaTest do
 
     # Cleanup!
     teardown_local_and_remote(org_name, repo_name)
+    Git.branch(Gitea.Helpers.local_git_repo(org_name, repo_name), ["-", repo_name])
```

Comment:
I don't think this command works as it seems the flag is missing: `["-", repo_name]`. Did you want to use `-D`?
Also `teardown_local_and_remote` delete the local folder I don't think this git command can run.

We can add checks on other Git function calls to make sure everything work correctly
```elixir
{:ok, _} = Git...
```

Reply:
updated. Thanks.